spec/acceptance/cli/run_spec.rb

```diff
@@ -44,4 +45,14 @@
       expect(output).to include 'Running: 1.1.0'
     end
   end
+
+  context 'when one of the arguments contains spaces' do
+    it 'preserves those spaces without raising an error' do
+      assertion = -> {
+        run 'appraisal 1.0.0 ruby -rbundler/setup -rdummy "test with spaces.rb"'
+      }
+
+      expect(&assertion).not_to raise_error
```

Comment:
Can you add assertion for output as well? I think I care more that the command get executed and yield correct result than if we return zero exit status.